dotnet-sdk-8.yaml

```diff
@@ -0,0 +1,148 @@
+package:
+  name: dotnet-sdk-8-only
```

Comment:
the .yaml filename and the first package name must match. I'm not sure if you wanted this with or without "-only".

Reply:
can we require in the build deps a specific version of an apk package? if so, that would be rather easy.